readthedocs/search/tests/utils.py

```diff
@@ -12,11 +15,61 @@ def get_search_query_from_project_file(project_slug, page_num=0, data_type='titl
     html_file = HTMLFile.objects.filter(project__slug=project_slug).order_by('id')[page_num]
 
     file_data = html_file.processed_json
-    query_data = file_data[data_type]
+    query_data = file_data[data_type.split('.')[0]]
+
+    if data_type == 'title':
+
+        # uses first word of page title as query
+        query = query_data.split()[0]
+
+    elif data_type.startswith('sections'):
+
+        # generates query from section title
+        if data_type.endswith('title'):
```

Comment:
why are we using endswith and startswith here, instead of just string checking?

Reply:
Sounds more pythonic.
I will update the PR.